### PR TITLE
Homework7: Ekaterina Zavadskaya

### PR DESCRIPTION
**Pull request by KateZavadskaya**
Add homework7

Docs: hm7_1.py - hm7_3.py

### CODE REVIEW

**Comment by KateZavadskaya** [comment]
@Stml89 added hm7, actions are sucessfull, check it, please 🌸.

File: hm7_1.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+""" Task 1: bulls and cows """
+
+x = input("Введите неповторяющееся четырёхзначное число: ")
```
Comment (Stml89) [inline hm7_1.py]: pls add checks:
1) it should be a number
2) only unique numbers are accepted

File: hm7_1.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+while True:
+    y = input("Что было загадано?: ")
```
Comment (Stml89) [inline hm7_1.py]: pls add checks:
1) it should be a number
2) only unique numbers are accepted

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hm7_1.py (hunk 1 above)
Comment (KateZavadskaya) [reply hm7_1.py]: @Stml89 done 👍, please

File: hm7_1.py (hunk 2 above)
Comment (KateZavadskaya) [reply hm7_1.py]: @Stml89 done 👍, please

**Review by Stml89 — APPROVED** [review]

**Pull request merged**